main/src/main/java/org/mobilitydata/gtfsvalidator/validator/MissingTripEdgeValidator.java

```diff
@@ -54,31 +55,35 @@ public void validate(NoticeContainer noticeContainer) {
                         tripFirstStop.csvRowNumber(),
                         tripFirstStop.stopSequence(),
                         trip.tripId(),
-                        "arrival_time"));
+                        "arrival_time",
+                        SeverityLevel.ERROR));
               }
               if (!tripFirstStop.hasDepartureTime()) {
                 noticeContainer.addValidationNotice(
                     new MissingTripEdgeNotice(
                         tripFirstStop.csvRowNumber(),
                         tripFirstStop.stopSequence(),
                         trip.tripId(),
-                        "departure_time"));
+                        "departure_time",
+                        SeverityLevel.WARNING));
```

Comment:
Overall the design here for rules that could be warnings or errors in different context is good, although IMHO MissingTripEdgeValidator isn't one of them.

http://gtfs.org/reference/static/#stop_timestxt says:

>An arrival time must be specified for the first and the last stop in a trip.

...but earlier it also says:

>If there are not separate times for arrival and departure at a stop, enter the same value for arrival_time and departure_time.

So in all cases all trips must have both arrival and departures times for the first and last record, or it's an ERROR.

Reply:
I removed these changes from this PR so that we can focus on the design rather than making the distinction for each particular notice. I'll open a new PR to discuss this. 

So in all cases all trips must have both arrival and departures times for the first and last record, or it's an ERROR.

> >If there are not separate times for arrival and departure at a stop, enter the same value for arrival_time and departure_time.
> 
> So in all cases all trips must have both arrival and departures times for the first and last record, or it's an ERROR.
> 
> 

Yes - I missed that one, I agree it should be an ERROR in this case. Thanks :)